docs/platform/concepts/support-case-and-incident-investigations.rst

```diff
@@ -0,0 +1,46 @@
+Support Case and Incident Investigations
+========================================
+
+Please provide the following information to the Aiven support team
+during a support case or on-going incident. Please copy and paste the
+following section as a template for your support/incident request. This
+give the Aiven team all the needed information to fix your issue, start
+investigations, or ask further questions.
+
+::
+
+   Scope of Impact
+   - Affected Projects:
+   - Affected Service Names: [If applicable]
+   - Affected Services: [Kafka, Postgres, Redis, Aiven Console, etc.]
+   - Affected Clouds/Regions: [AWS, google-us-east1, etc.]
+   - Affected Functionality: [networking, deployments, metrics, etc]
+
+   Severity
+   - Clients can connect: [Yes/No/Partial Failure]
+   - Service is responsive: [Yes/No/Partial Failure]
+
+   Additional Details
+
+   - What is expected behavior?
+   - What is the actual behavior?
+   - What languages/frameworks/versions are your using?
+   - Can you share client side error messages?
+   - Is there a history of this issue/behavior?
+   - Can you reproduce the behavior? [Yes/No] 
+     If Yes please describe the steps to reproduce it.
+
+Communication
+-------------
+
+The support team will be in contact with you for all open support cases.
+Please see `Aiven's support
+SLA's <https://help.aiven.io/en/articles/868101-aiven-support-details>`__
```

Comment:
Please use the updated link (where this one redirects to) and change the link wording to just say "support details" or some other valid word to tackle this error